cluster/saltbase/salt/README.md

```diff
@@ -0,0 +1,31 @@
+This directory forms the base of the main SaltStack configuration. The
+place to start with any SaltStack configuration is
+[top.sls](top.sls). However, unless you are particularly keen on
+reading Jinja templates, the following tables break down what
+configurations run on what providers:
+
+Key: M = Config applies to master, n = config applies to nodes
+
+Config                                              | GCE   | Vagrant | AWS | Azure
+----------------------------------------------------|-------|---------|-----|------
+[cadvisor](cadvisor/)                               | M n   | M n     | M n | M n
+[debian-auto-upgrades](debian-auto-upgrades/)       | M n   | M n     | M n | M n
+[docker](docker/)                                   | M n   | M n     | M n |   n
+[etcd](etcd/)                                       | M     | M       | M   | M
+[fluentd-es](fluentd-gcp/) (pillar conditional)     | M n   | M n     | M n | M n
+[fluentd-gcp](fluentd-gcp/) (pillar conditional)    | M n   | M n     | M n | M n
+[generate-cert](generate-cert/)                     | M     | M       | M   | M
+[kube-proxy](kube-proxy/)                           |   n   |   n     |   n |   n
+[kube-addons](kube-addons/)                         | M     | M       | M   | M
+[kube-apiserver](kube-apiserver/)                   | M     | M       | M   | M
+[kube-controller-manager](kube-controller-manager/) | M     | M       | M   | M
+[kube-client-tools](kube-client-tools/)             | M     | M       | M   | M
+[kube-scheduler](kube-scheduler/)                   | M     | M       | M   | M
+[kubelet](kubelet/)                                 | M n   | M n     | M n |   n
+[logrotate](logrotate/)                             | M n   |   n     | M n | M n
+[monit](monit/)                                     | M n   | M n     | M n | M n
+[nginx](nginx/)                                     | M     | M       | M   | M
+[openvpn](openvpn/)                                 |       |         |     | M
+[openvpn-client](openvpn-client/)                   |       |         |     |   n
```

Comment:
nit: giuthub has `-client` first. Whether or not it's right, it'd be nice to fully match the ordering.